### DISCUSSION TITLE
customtube copy???

### DISCUSSION THREAD

**Discussion by bangbangyouall** [lightbeam24/CustomTube#108] [General]
there is this add-on called 'Tube Customize". it is on 1.0.4.4 & the description is "Tailor your YouTube experience with the CustomTube extension. Choose from a collection of recreated layouts, including various YouTube designs from 2012-2017 and 2019. Discover the perfect interface that aligns with your preferences, allowing for a personalized and visually pleasing browsing experience.", i think this is a red flag.

link: https://addons.mozilla.org/en-US/firefox/addon/tube-customize/

**Comment by lightbeam24** [comment]
Yep, an exact copy of 1.0.4.4, but around 1KB bigger, and I'll bet that extra 1KB is probably some tracking code.

It looks like there are several copies of CustomTube on AMO, all of which are funnily enough on 1.0.4.4, even if they released after 1.1.0 came out. I don't know if I should be angry or honoured that I made an extension good enough to be copied like this lol. I guess I'll try reporting them.
